src/Aether/Aether.fs

```diff
@@ -110,39 +110,39 @@ module Compose =
 
 (* Lenses
 
-   Various lenses implemented for common types such as tuples,
-   lists and maps, along with an id lens (which is useful for composing
-   a lens which has not specific "lensing" elements but is implicitly a chain
-   of one or more isomorphisms. Having an id lens enables the root composition. *)
+   Various lenses for common types such as tuples,
+   lists, and maps; along with an id_ lens (which is useful for composing
```

Comment:
Your welcome

Indeed. Upon reflection, I am thinking that a Lens is just a complementary pair of isomorphisms (had to look up that word :)) or lossless type converters. Consequently, a better terminology for the pair might have been `(from, to)` instead of `(get, set)` which is more in keeping with immutable data structures. Additionally, I was surprise that the operator `setPartialLens` does not allow for None as a value.